universal-application-tool-0.0.1/app/views/admin/AdminLayout.java

```diff
@@ -26,6 +24,35 @@ public AdminLayout(ViewUtils viewUtils) {
     super(viewUtils);
   }
 
+  @Override
+  public HtmlBundle getBundle() {
+    return getBundle(new HtmlBundle());
+  }
```

Comment:
Same question as in ApplicantLayout: Do we need to override this?

Reply:
What's the verdict here? We need to override it?

Reply:
This should be removed with my latest changes.